.github/scripts/run_nm.py

```diff
@@ -0,0 +1,167 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
```

Comment:
perhaps we could use bloaty?

Reply:
yeah I did think about it. This is just `nm` TBH so not sure if we need it to be more complicated than that. Wrote a python wrapper to print sorted table, and use cross-tools. But `bloaty` is nice though a bit "bloated" with dependency etc.